validators.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	v1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	log "github.com/sirupsen/logrus"
+)
+
```

Comment:
Can we alias the `func(v1.Job) bool` to something like `JobValidator`. 
```Go
type JobValidator func(v1.Job) bool
```

Reply:
Addressed in https://github.com/theMagicalKarp/kube-janitor/pull/6/commits/d7db5c0f2cc7a69d8332beff36489d9456fa6c80